haystack/agents/base.py

```diff
@@ -82,12 +86,12 @@ def __init__(
          Creates an Agent instance.
 
         :param prompt_node: The PromptNode that the Agent uses to decide which tool to use and what input to provide to it in each iteration.
-        :param prompt_template: The name of a PromptTemplate supported by the PromptNode or a new PromptTemplate. It is used for generating thoughts and running Tools to answer queries step-by-step.
-        :param tools: A List of Tools that the Agent can choose to run. If no Tools are provided, they need to be added with `add_tool()` before you can use the Agent.
-        :param max_iterations: The number of times the Agent can run a tool plus then once infer it knows the final answer.
-            Set at least to 2 so that the Agent can run one Tool and then infer it knows the final answer. Default 5.
-        :param tool_pattern: A regular expression to extract the name of the Tool and the corresponding input from the text generated by the Agent.
-        :param final_answer_pattern: A regular expression to extract final answer from the text generated by the Agent.
+        :param prompt_template: The name of a PromptTemplate for the PromptNode. It's used for generating thoughts and running tools to answer queries step-by-step. You can use the default `zero-shot-react` template or create a new template in a similar format.
```

Comment:
`running tool` might be a bit misleading here. The template is only used to generate the name of the tool and its input. I'd suggest `choosing` instead.